### PR TITLE
User product guide

### PR DESCRIPTION
**Pull request by MagdalenaZuba**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | https://issues.ibexa.co/browse/IBX-6498
| Versions      | master
| Edition       | n/a

#### Checklist

- [ ] Text renders correctly
- [ ] Text has been checked with vale
- [ ] Description metadata is up to date
- [ ] Redirects cover removed/moved pages
- [ ] Code samples are working
- [ ] PHP code samples have been fixed with PHP CS fixer
- [ ] Added link to this PR in relevant JIRA ticket or code PR

### CODE REVIEW

File: docs/users/user_management_guide.md [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+[[= product_name_base =]] offers flexibility in authentication methods to cater to different user bases and security requirements.
+
+The system's OAuth integration allows users to log in with their existing social media or third-party credentials, supporting OAuth 2.0 for easy integration with services like Google, Facebook, and Twitter.
+
+![Log in via Google](log_in_via_google.png)
+
+Available options:
+
+- [Username and password](user_management.md) - ideal for most users, this traditional method offers a secure login process with username and password.
+- [OAuth](oauth_authentication.md) - integrating OAuth authentication allows users to log in using their existing social media or third-party credentials, streamlining the registration process.
```
Comment (juskora) [inline docs/users/user_management_guide.md:89]: I would rewrite it a little bit as it sounds like repetition from line: 82.

File: docs/users/user_management_guide.md [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+
+The [invitation system](invitations.md) streamlines user onboarding and engagement.
+Track the status of invitations, including when they were sent, whether they were accepted, and the actions taken by users who accepted them.
+
+![Invitations](users_invitation.png)
+
+### User segmentation and personalization
+
+[[= product_name_base =]]'s segmentation and personalization features allow organizations to deliver customized user experiences.
+Track user behavior, such as page views, search queries, and interactions, to create Segments and Segment Groups for users who share similar actions.
```
Comment (juskora) [inline docs/users/user_management_guide.md:102]:
```suggestion
Track user behavior, such as page views, search queries, and interactions, to create Segments and Segment Groups for users who share similar behaviors.
```

File: docs/users/user_management_guide.md [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+Possible uses:
+
+- Demographics - Segment users based on demographic data such as age, location, and gender to personalize content, promotions, and recommendations.
+- Behavior - Tailor content based on user behavior, such as frequent content consumption, shopping patterns, or search history, ensuring users see what they're interested in.
+- Preferences - Utilize user preferences to offer a customized experience, from language preferences to Content Type preferences.
+
+### Customer groups
+
+The customer group functionality allows for targeted content delivery and service offerings.
+Set specific permissions for customer groups to control who can access and edit certain content ot get specific recommendations.
```
Comment (juskora) [inline docs/users/user_management_guide.md:115]:
```suggestion
Set specific permissions for customer groups to control who can access and edit certain content to get respective recommendations.
```

File: docs/users/user_management_guide.md [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+
+Possible uses:
+
+- product recommendations - Create customer groups based on product preferences and offer tailored product recommendations.
+
+- targeted marketing campaigns - Segment users by demographics or behavior and deliver targeted marketing campaigns that resonate with specific groups.
```
Comment (juskora) [inline docs/users/user_management_guide.md:121]: Rewrite as it resembles lines: 108.

File: docs/users/user_management_guide.md [hunk 5]
```diff
@@ -0,0 +1,159 @@
...
+
+- Roles and Permissions - define Roles and assign Permissions to them. This ensures that users have appropriate access to content and functionalities. Roles can be customized to match the organization's specific needs.
+
+- authentication methods - enable multiple authentication methods, including traditional username and password, OAuth, and external service logins. This flexibility allows organizations to adapt to various user authentication requirements.
+
+- user segmentation - Segment users based on criteria such as demographics, behavior, or preferences. This segmentation enables personalized content delivery and targeted marketing.
+
+- invitations - invite users to join a platform. This is useful for onboarding new users or sending invitations for exclusive content or events.
+
+- customer groups - organize users into customer groups, which helps in delivering tailored experiences and content to specific segments.
```
Comment (juskora) [inline docs/users/user_management_guide.md:39]: How about to begin with capital letter?

File: docs/users/user_management_guide.md [hunk 6]
```diff
@@ -0,0 +1,159 @@
...
+[[= product_name_base =]]'s segmentation and personalization features allow organizations to deliver customized user experiences.
+Track user behavior, such as page views, search queries, and interactions, to create Segments and Segment Groups for users who share similar actions.
+
+![Segment Groups](../administration/img/admin_panel_segment_groups.png)
+
+Possible uses:
+
+- Demographics - Segment users based on demographic data such as age, location, and gender to personalize content, promotions, and recommendations.
+- Behavior - Tailor content based on user behavior, such as frequent content consumption, shopping patterns, or search history, ensuring users see what they're interested in.
+- Preferences - Utilize user preferences to offer a customized experience, from language preferences to Content Type preferences.
```
Comment (juskora) [inline docs/users/user_management_guide.md:110]: Begin with capital letter, after hyphen, lower case.

File: docs/users/user_management_guide.md (hunk 4 above)
Comment (juskora) [inline docs/users/user_management_guide.md:121]: Capital letter -> - -> lower case.

**Review by juskora — CHANGES_REQUESTED** [review]

**Review by juskora — APPROVED** [review]

File: docs/users/user_management_guide.md [hunk 7]
```diff
@@ -0,0 +1,154 @@
...
+and efficient user experiences while ensuring that user access and content delivery align with their business goals and strategies.
+
+### User Roles and Permissions
+
+[[= product_name_base =]] allows you to define custom user [Roles with granular Permissions](permission_overview.md), ensuring that users have access to only the specific parts of the system they need.
+Furthermore, you can create user groups to simplify Permission management.
+Assign multiple users to a group to ensure consistency and ease of access control.
+This helps maintain effortless security and control.
+
+**Permission:**
```
Comment (julitafalcondusza) [inline docs/users/user_management_guide.md:56]: Just an option to consider - list is more clear (in my opinion), and additionally, in the current form, when each sentence begins with "A" letter ('A Permission...', 'A Role...'), I see it as a letter numbered list.

Option:

- **Permission** - defines a specific action or access level that can be granted or denied within the system.

- **Role** - represents a collection of Permissions that can be assigned to users or user groups.
Roles streamline permission management by grouping related Permissions together.

- **Policy** - is a set of rules or conditions that determine under what circumstances a specific permission is granted or denied by applying Limitations.
Policies allow for fine-grained control of access based on various factors, such as user attributes or system states.

**Review by julitafalcondusza — APPROVED** [review]

File: docs/users/user_management_guide.md [hunk 8]
```diff
@@ -0,0 +1,147 @@
...
+
+User management is available in all [[= product_name =]] versions.
+
+## How does user management work
+
+[[= product_name =]] simplifies user management with an intuitive and powerful system of accounts, Roles, Permissions, groups, and Segments.
+You can find all user groups and Users in the **Admin Panel** by selecting **Users**.
+Here, you can manage users, their relations, Roles, and Policies.
+
+![User's section](users_section.png)
```
Comment (dabrt) [inline docs/users/user_management_guide.md:25]:
```suggestion
![Users section](users_section.png "Users section")
```

File: docs/users/user_management_guide.md [hunk 9]
```diff
@@ -0,0 +1,147 @@
...
+
+- User accounts - create and manage user accounts. This includes capturing user information, such as name, email, and profile details.
+
+- Roles and Permissions - define Roles and assign Permissions to them. This ensures that users have appropriate access to content and functionalities. Roles can be customized to match the organization's specific needs.
+
+- Authentication methods - enable multiple authentication methods, including traditional username and password, OAuth, and external service logins. This flexibility allows organizations to adapt to various user authentication requirements.
+
+- User segmentation - segment users based on criteria such as demographics, behavior, or preferences. This segmentation enables personalized content delivery and targeted marketing.
+
+- Invitations - invite users to join a platform streamlining an onboarding process, sending invitations for exclusive content or events.
```
Comment (dabrt) [inline docs/users/user_management_guide.md:37]:
```suggestion
- Invitations - invite users to join a platform, to streamline an onboarding process, or send exclusive content or event invitations.
```

File: docs/users/user_management_guide.md [hunk 10]
```diff
@@ -0,0 +1,147 @@
...
+To help you understand further the role each element serves, here's a brief summary: 
+
+- Role - represents a collection of Permissions that can be assigned to users or user groups. Roles streamline permission management by grouping related Permissions together.
+- Permission - defines a specific action or access level that can be granted or denied within the system.
+- Policy - is a set of rules or conditions that determine under what circumstances a specific permission is granted or denied by applying Limitations.
+Policies allow for fine-grained control of access based on various factors, such as user attributes or system states.
+
+### Custom Policies
+
+[Tailor user access control](custom_policies.md) to your unique requirements using custom policies. Define complex rules and access criteria for different users or groups.
```
Comment (dabrt) [inline docs/users/user_management_guide.md:65]:
```suggestion
[Tailor user access control](custom_policies.md) to your unique requirements by creating custom policies. Define complex rules and access criteria for different users or groups.
```

**Pull request merged**